cycle-app.py

```diff
@@ -0,0 +1,270 @@
+import dash
+import dash_core_components as dcc
+import dash_html_components as html
+import dash_table as tbl
+from dash.dependencies import Input, Output, State
+
+from collections import OrderedDict
+import os
+import base64
+import io
+
+import bicycleparameters as bp
+import pandas as pd
+import numpy as np
+
+###!!!Must ensure that current working directory is set to directory immediatly containing '\data\'!!!###
+###---------------------------------------------------------------------------------------------------###
+
+# Initializes bike data as Bicycle and retrieves its parameters
+
+
+def new_par(bike_name):
+    bike = bp.Bicycle(bike_name, pathToData=os.getcwd()+'\\data')
```

Comment:
I think that the data directory is installed with bicycle parameters. A better way to locate this, is relative to this file. But I think the cycle-app.py file needs to be moved inside the bicycleparameters/ directory first, so that it will be packaged along with the other files.